apps/item_lock/components/item_lock.py

```diff
@@ -108,6 +111,13 @@ def unlock(self, item_filter, user_id, session_id, etag):
                 superdesk.get_resource_service('archive').delete_action(lookup={'_id': item['_id']})
                 push_content_notification([item])
             else:
+                autosave = superdesk.get_resource_service('archive_autosave').find_one(req=None, _id=item['_id'])
+                if autosave:
+                    autosave['force_unlock'] = True  # to avoid item locked check
+                    resolve_document_version(autosave, 'archive', 'PATCH', item)
+                    superdesk.get_resource_service('archive').patch(item['_id'], autosave)
```

Comment:
There is a workflow transition check on patch call. Can you just check that this work for `correct` and `kill` action?